.github/workflows/build.yml

```diff
@@ -13,41 +10,57 @@ jobs:
   build-el8stream:
     runs-on: [ self-hosted, image-builders ]
     if: |
-      github.event.issue.pull_request &&
-      startsWith(github.event.comment.body, '/ci') &&
-      (
-        github.event.comment.author_association == 'MEMBER' ||
-        github.event.comment.author_association == 'COLLABORATOR'
+      github.event_name == 'workflow_dispatch' ||
+      ( github.event.issue.pull_request &&
+        startsWith(github.event.comment.body, '/ci') &&
+        (
+          github.event.comment.author_association == 'MEMBER' ||
+          github.event.comment.author_association == 'COLLABORATOR'
+        )
       )
     container:
       image: quay.io/ovirt/buildcontainer:el9stream
-      options: --group-add qemu --user build --workdir /home/build ---privileged
+      options: --group-add qemu --user build --privileged
 
     steps:
 
-    - uses: actions/checkout@v2
+    - name: Checkout
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        cd /home/build
+        gh repo clone oVirt/ost-images -- -n --depth 1
+        cd ost-images
+        git fetch --depth 1 origin $GITHUB_REF
+        git checkout HEAD
```

Comment:
Don't we want `FETCH_HEAD` here instead?

Reply:
right...